index.html

```diff
@@ -100,11 +100,11 @@ <h3 id="headline">Here's a time line of Dr. Borlaug's life:</h3>
         <blockquote
           cite="http://news.rediff.com/report/2009/sep/14/pm-pays-tribute-to-father-of-green-revolution-borlaug.htm"
         >
-          <p>
+          <h1>
```

Comment:
you add a text to give more to read for the reader which is a decent job by itself this is the article that the user reads or the content of the paper so it doesn't need any special attention, if we enclose this text in  p  tag instead of  h1  tag more semantically decent look doing this also makes life harder for those people who depend on assistive technology to read your article so consider this as well mate.